### PR TITLE
Expire the Notice when post transitions statuses

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#62408](https://central.tri.be/issues/62408)
_Rel:_ moderntribe/tribe-common#130

### CODE REVIEW

File: src/Tribe/Main.php [hunk 1]
```diff
@@ -574,13 +574,57 @@ protected function addHooks() {
...
+		 * Displays the Archive confict notice using Tribe__Admin__Notices code
+		 *
+		 * @return string
+		 */
 		public function render_notice_archive_slug_conflict() {
 			$archive_slug = Tribe__Settings_Manager::get_option( 'eventsSlug', 'events' );
 			$conflict     = get_page_by_path( $archive_slug );
 
-			if ( ! $conflict || 'trash' === $conflict->post_status ) {
+			var_dump( $conflict );
```
Comment (barryhughes) [inline src/Tribe/Main.php]: Leftover from testing?

Comment (bordoni) [inline src/Tribe/Main.php]: OOpsi!

**Comment by bordoni** [comment]
Rdy for your Code Review 👀

**Comment by barryhughes** [comment]
:100:

**Pull request merged**